performance/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
-if(BUILD_SLOWDOWN)
-    add_subdirectory(test)
-endif()
+add_subdirectory(test)
+
+add_library(concord_resources_monitor INTERFACE)
+
+target_include_directories(concord_resources_monitor INTERFACE
+  include
+)
```

Comment:
I think the intention for the `performance` directory was to have the performance testing framework inside. Not sure if it is the right place for the resources monitor. Maybe we can have it in a separate directory? What do you think?